src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -185,6 +185,49 @@ public static string DownloadTensorFlowSentimentModel()
             return path;
         }
 
+        public static string DownloadTaxiFareData()
+        {
+            string githubPath = "https://raw.githubusercontent.com/dotnet/machinelearning-samples/master/samples/csharp/getting-started/Regression_TaxiFarePrediction/TaxiFarePrediction/Data/taxi-fare-train.csv";
```

Comment:
We already have this file in the repo. See

https://github.com/dotnet/machinelearning/blob/37edde9d839d3bd14e57666c2dc634846eb7fd0d/test/Microsoft.ML.Tests/FeatureContributionTests.cs#L417

https://github.com/dotnet/machinelearning/blob/37edde9d839d3bd14e57666c2dc634846eb7fd0d/test/Microsoft.ML.Tests/FeatureContributionTests.cs#L388-L417

We shouldn't need to download the file again.